internal/cmd/cmd_set.go

```
@@ -11,7 +11,8 @@
 	"github.com/dicedb/dice/internal/object"
 	"github.com/dicedb/dice/internal/server/utils"
 	"github.com/dicedb/dice/internal/shardmanager"
+	"github.com/dicedb/dice/internal/store"
```

golangci-lint (1.64.8)

ST1019: package "github.com/dicedb/dice/internal/store" is being imported more than once

(stylecheck)